code/modules/clothing/under/jobs/security.dm

```diff
@@ -18,6 +18,15 @@
 	siemens_coefficient = 0.9
 	rolled_sleeves = 0
 
+/obj/item/clothing/under/rank/warden/skirt
+	desc = "Standard feminine fashion for a Warden. It is made of sturdier material than standard jumpskirts. It has the word \"Warden\" written on the shoulders."
+	name = "warden's jumpskirt"
+	icon_state = "wardenf"
+	item_state = "r_suit"
+	worn_state = "wardenf"
+	armor = list(melee = 10, bullet = 0, laser = 0,energy = 0, bomb = 0, bio = 0, rad = 0)
+	siemens_coefficient = 0.9

```

Comment:
If it's a subtype of the standard warden's jumpsuit, then you don't need to specify the armor/siemens values. In fact, you probably shouldn't, so that they stay the same if the basic needs changing. Same deal with the other two.